src/processors/models/database/cassandra/cassandraConnector.js

```diff
@@ -18,6 +20,11 @@ async function init(cassandraClient) {
     client = cassandraClient;
 }
 
+async function getAllProcessors(from, limit) {
+    const resultRows = await executeQuery(GET_ALL_PROCESSORS, {}, {});
```

Comment:
change to `const resultRows = await executeQuery(GET_ALL_PROCESSORS, [], queryParams);`